src/styles/blocks/card.scss

```diff
@@ -0,0 +1,128 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: $white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:hover {
+    transform: scale(120%);
```

Comment:
The `transform: scale(120%)` should be `transform: scale(1.2)` to ensure proper scaling. The scale function takes a multiplier, not a percentage.